src/index.js

```diff
@@ -298,30 +329,33 @@ const createGlobalHeader = ({ headerTitle, button1Label, button1OnClick, button2
     }, headerElement)
 }
 
-const viewContact = () => {
+const viewContact = (person) => {
```

Comment:
function name is not accurate => view is not something JS can do. Maybe `renderContact`?